tests/tt_eager/python_api_testing/unit_testing/misc/test_sharded.py

```diff
@@ -1975,7 +1977,8 @@ def test_reshard(
     output_shard_grid = ttl.tensor.CoreRangeSet({ttl.tensor.CoreRange(ttl.tensor.CoreCoord(0, 0), compute_grid)})
     output_shard_spec = ttl.tensor.ShardSpec(output_shard_grid, output_shard_shape, output_shard_orientation, False)
     output_mem_config = ttl.tensor.MemoryConfig(output_sharding_scheme, ttl.tensor.BufferType.L1, output_shard_spec)
-    tt_dtype = ttl.tensor.DataType.BFLOAT16
+    if input_layout == ttl.tensor.Layout.ROW_MAJOR and tt_dtype == ttl.tensor.DataType.BFLOAT8_B:
+        pytest.skip("Illegal layout/dtype config")
```

Comment:
If this combo is illegal, let's add this check to the op's validate method.

Reply:
This is not a problem with the op, it's that BFP8 doesn't work/exist for RM layout. It is a tile layout only dtype